core/javax.cache/src/main/java/org/wso2/carbon/caching/impl/CarbonCacheManager.java

```diff
@@ -202,13 +203,13 @@ boolean removeLocalCaches() {
         return this.caches.isEmpty();
     }
 
-    @Override
-    public javax.transaction.UserTransaction getUserTransaction() {
-        Util.checkAccess(ownerTenantDomain, ownerTenantId);
-        checkStatusStarted();
-        touch();
-        return null;  //To change body of implemented methods use File | Settings | File Templates.
-    }
+//    @Override
```

Comment:
`javax.transaction` is not available with JDK 17. Since this seems to be not used, removed. But need to validate the functionality.